src/test/behave_tests/features/steps/cephadm_test.py

```diff
@@ -0,0 +1,28 @@
+import logging
+
+from behave import *
+from kcli_handler import exec_ssh_cmd
+
+
+@given('I log as root into {node} and I execute')
+def init_step(context, node):
+    context.node = node
+    commands = context.text.split('\n')
+    for command in commands:
+        exec_ssh_cmd(context.node, command)
+    
+
+@when('I execute')
+def execute_step(context):
+    context.output = exec_ssh_cmd(
+        context.node,
+        context.text
+    )
+    logging.info(f"exec output : {context.output}")
+
+@then('I get')
```

Comment:
Improve this step ... 
@then('I get a result witch contains:')

The tesxt provided in the behave text must be included as part of the command output